packages/terra-abstract-modal/src/AbstractModal.jsx

```diff
@@ -154,7 +154,7 @@ class AbstractModal extends React.Component {
     }
 
     setTimeout(() => {
-      if (this.state.modalTrigger) {
+      if (this.state.modalTrigger && this.state.modalTrigger.focus) {
```

Comment:
This is quite similar to https://github.com/davidtheclark/focus-trap-react/pull/19